src/authn/ccloudProvider.ts

```diff
@@ -642,46 +632,45 @@ export class ConfluentCloudAuthProvider
    * Handle the URI event for the authentication callback.
    * @param uri The URI that was handled.
    */
-  async handleUri(uri: vscode.Uri): Promise<void> {
-    if (uri.path === "/authCallback") {
```

Comment:
The only real change here is removing the `.path` check since the UriEventHandler does that in order to fire the `ccloudAuthCallback` event emitter